client/store/assessment/hooks/index.ts

```diff
@@ -0,0 +1,5 @@
+import { useAppSelector } from '@client/store'
+import { AssessmentName } from '@core/meta/assessment'
+
+export const useAssessmentPropsName = (): AssessmentName =>
```

Comment:
better useAssessment and that's it.
then in the component you use assessment.props.name